src/main/resources/resources/GroupNameResources_pt.properties

```diff
@@ -23,7 +23,9 @@ label.name.AlumniGroup = Alumni{0}
 label.name.CampusEmployeeGroup = Funcion�rios do campus {0}
 label.name.CandidateGroup = Candidatos
 label.name.CerimonyInquiryGroup = Pessoas alvo do inqu�rito {0}
-label.name.CoordinatorGroup = Coordena��o{0}{1}
+label.name.CoordinatorGroup= Coordena��o{0}{1}{2}
+label.name.CoordinatorGroup.responsible.false= \u0020Adjunta
```

Comment:
Is this file latin1 ? it should be .. please remove \u0020 character.

Reply:
It's ISO-8859. There are other occurences of \u0020 in this file which is why I used this for explicit whitespace.